conviva/README.md

```diff
@@ -26,26 +26,28 @@ This monitor requires:
 
 ### INSTALLATION
 
-Install the latest version of the SignalFx Smart Agent as described [here](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
+Install the latest version of the SignalFx Smart Agent as described in [Installation](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
 
 ### CONFIGURATION
 
-Find and edit the SignalFx Smart Agent configuration file `agent.yaml` to configure the conviva monitor as described <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md">here</a>. Use the metric parameters <a target="_blank" href="https://github.com/signalfx/integrations/blob/master/conviva/docs/conviva_metrics.md">here</a> as values for `metricParameter`.
+1. Open the SignalFX Smart Agent `agent.yaml`configuration file.
+2. Edit the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#overview">`monitors`</a> section.
+3. Add the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#conviva-monitor-metric-parameters-and-metrics">metric parameters</a> as values for `metricParameter`.
```

Comment:
Link to the doc site instead of to GitHub:

https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html#conviva-monitor-metric-parameters-and-metrics